frontend/isoko/src/features/dashboard/DashboardAPI.ts

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import { environment } from '../../environment/environment';
+import { Business } from '../../types/GlobalTypes';
+
+export const fetchBusinessDetails = async (
```

Comment:
Since this is the same function as the one from the businessAPI, should we refactor to reduce duplicate code? Maybe combine these 2 API files together or import the other one into this file.

Reply:
Yeah good point, I'll refactor to use the same code for both